### DISCUSSION TITLE
What happened to Inspector? No Visual Tree debugging?

### DISCUSSION THREAD

**Discussion by EmilAlipiev** [xamarin/Xamarin.Forms#11959] [General]
Where is inspector in Enterprise version. I thought it is one of the enterprise paid tool but not exist any longer in VS 2019.
How can we make visual tree inspection otherwise? is there any other tool?

**Comment by Reveon** [comment]
Issue was [created](https://developercommunity.visualstudio.com/content/problem/673735/no-xamarin-inspector-option-in-vs2019.html) over a year ago, ignored for the same amount of time and then just a small comment
"**Inspector is no longer part of Visual Studio.**" without this being mentioned in the the official [docs](https://docs.microsoft.com/en-us/xamarin/tools/inspector/)

I am starting to get a feeling that this is now a standard MS approach for all Xamarin issues.
